packages/terra-clinical-result/src/ClinicalResultBloodPressure.jsx

```diff
@@ -0,0 +1,292 @@
+     */
+    diastolic: observationPropShape,
+  }),
+  /**
+   * Visually hides the unit of measure when presented in a series of side-by-side columns of the same unit.
+   */
+  hideUnit: PropTypes.bool,
+  /**
+   * Whether or not the text should be truncated in display. Restricts clinical result details each to one line.
+   */
+  isTruncated: PropTypes.bool,
+  /**
+   * Override that shows an Error display. Used when there is a known error or problem when retrieving or assembling the clinical result data.
+   */
+  hasResultError: PropTypes.bool,
+  /**
+   * Override that shows a known "No Data" display. Used when there is known to be no value for a given clinical result concept at a specific datetime.
+   */
+  hasResultNoData: PropTypes.bool,
+  /**
+   * @private
+   * The intl object to be injected for translations.
+   */
+  intl: intlShape.isRequired,
+};
+
+const defaultProps = {
+  resultData: {},
+  hideUnit: false,
+  isTruncated: false,
+  hasResultError: false,
+  hasResultNoData: false,
+};
+
+const ClinicalResultBloodPressure = (props) => {
+  const {
+    resultData,
+    hideUnit,
+    isTruncated,
+    hasResultError,
+    hasResultNoData,
+    intl,
+    ...customProps
+  } = props;
+
+  let clinicalResultBloodPressureDisplay = null;
+
+  if (hasResultError) {
+    clinicalResultBloodPressureDisplay = <ResultError />;
+  } else if (hasResultNoData) {
+    clinicalResultBloodPressureDisplay = <NoData />;
+  } else {
+    const CompareTemplate = (s = null, d = null) => ({
+      systolic: s,
+      diastolic: d,
+    });
+    const compareUnits = new CompareTemplate();
+    const compareConceptDisplays = new CompareTemplate();
+    const compareDatetimeDisplays = new CompareTemplate();
+    const compareStatusIsInError = new CompareTemplate();
+
+    let hasModifiedIcon = false;
+    let hasCommentIcon = false;
+    let hasUnverifiedIcon = false;
+
+    let iconGroupDisplayElement = null;
+    let conceptDisplayElement = null;
+    let datetimeDisplayElement = null;
+    const decoratedResultDisplay = [];
+
+    const hasSystolic = resultData.systolic;
+    const noDataSystolic = (hasSystolic && resultData.systolic.resultNoData === true);
+    if (hasSystolic && !noDataSystolic) {
+      const {
+        result = null,
+        status = null,
+        conceptDisplay = null,
+        datetimeDisplay = null,
+        isModified = null,
+        hasComment = null,
+        isUnverified = null,
+      } = hasSystolic;
+
+      if (!isEmpty(result.unit)) { compareUnits.systolic = result.unit.trim().toLowerCase(); }
+      if (!isEmpty(status)) { compareStatusIsInError.systolic = checkIsStatusInError(hasSystolic); }
+      if (!isEmpty(conceptDisplay)) { compareConceptDisplays.systolic = conceptDisplay.trim().toLowerCase(); }
+      if (!isEmpty(datetimeDisplay)) { compareDatetimeDisplays.systolic = datetimeDisplay.trim().toLowerCase(); }
+      if (!isEmpty(isModified)) { hasModifiedIcon = isModified; }
```

Comment:
Arn't `isModified`, `hasComment `, and `isUnverified` booleans? If yes, I don't think we need to check isEmpty since thats a string helper. If these arn't booleans, can we rename these so they don't represent a bool?